compiler/aarch64/codegen/ARM64SystemLinkage.hpp

```diff
@@ -116,6 +116,35 @@ class ARM64SystemLinkage : public TR::Linkage
     * @param[in] node : caller node
     */
    virtual TR::Register *buildIndirectDispatch(TR::Node *callNode);
+
+   /**
+    * @brief Provides the entry point in a method to use when that method is invoked
+    *        from a method compiled with the same linkage.
+    *
+    * @details
+    *    When asked on the method currently being compiled, this API will return 0 if
+    *    asked before code memory has been allocated.
+    *
+    *    The compiled method entry point may be the same as the interpreter entry point.
+    *
+    * @return The entry point for compiled methods to use; 0 if the entry point is unknown
+    */
+   virtual intptrj_t entryPointFromCompiledMethod();
```

Comment:
More of a general comment: what's the value of duplicating the comments in every override? It seems like these could easily get out of sync. That said, I can see some value in documenting override-specific behaviour.

Reply:
Perhaps we need some "standard" comment text to use when a project or architecture provides a specialization of a function in an extensible class (e.g., like an `@override` but in a comment).  This isn't something to be decided in this PR.